lib/websocket_proxy.rb

```diff
@@ -61,6 +61,10 @@ def write(string)
     @ws.write(string)
   end
 
+  def crash_report(ex)
+    "#{ex.class} for #{@console.vm_id}: #{ex.message}\n#{ex.backtrace.join("\n")}"
```

Comment:
Be really careful about `@console.vm_id`... we're in an exception handler, we shouldn't fail to log the message because `@console` is nil or doesn't respond_to `vm_id`.